<launch>
  <node pkg="darknet_rsl" type="darknet_rsl" name="darknet_rsl" output="screen">
    <param name="cfg_path" 		value="/home/markob/git/darknet/cfg/tiny-yolo-voc.cfg" />
    <param name="weights_path" 		value="$(find darknet_rsl)/weights/tiny-yolo-voc.weights" />
    <param name="data_path" 		value="/home/markob/git/darknet/data" />
    <param name="camera_topic_name" 	value="/camera/image_raw" />
    <param name="object_threshold" 	value="0.2" />
  </node>

  <!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>
